app/src/componentLibrary/autocompletes/multipleAutocomplete/multipleAutocomplete.jsx

```diff
@@ -251,6 +253,9 @@ MultipleAutocomplete.propTypes = {
   parseInputValueFn: PropTypes.func,
   handleUnStoredItemCb: PropTypes.func,
   dataAutomationId: PropTypes.string,
+  existingLaunchNames: PropTypes.shape({
```

Comment:
Should we use shape for 1 field ? And `existingLaunchNames` name is little bit confusing, because it sounds like it should be array of names, not object with 1 bool field. May be `isExistingLaunchNames' or it should be `string`, not `bool` ?

Reply:
Good point. But I think that we can use shape even for one field in object. And that object can contains a lot of fields. Every field consists of key:boolean.
Why I use this? Because already existing logic require us to use objects like this.
But isExistingLauncNames sounds better!